src/utils/webrtc/webrtc.js

```diff
@@ -293,6 +293,7 @@ function usersChanged(signaling, newUsers, disconnectedSessionIds) {
 				webRtc: webrtc,
 			})
 		}
+		callParticipantModel.setActor(user.actorType, user.actorId)
```

Comment:
desktop client goes :boom: when speaking to a 19 server?

Reply:
The attributes would be simply `undefined`, which would not change the previous behaviour nor break anything, no?

Reply:
ah, no PHP style `Undefined key X` error. fine by me then